.github/workflows/release.yml

```
@@ -39,7 +39,31 @@ jobs:
       - name: Build
         run: pnpm build
 
+      - name: Check for changes between main and changeset-release/main
+        id: check_changes
+        run: |
+          # :A: ctx Check if changeset-release/main branch exists and has differences
+          # Fetch the changeset-release/main branch if it exists
+          git fetch origin changeset-release/main:refs/remotes/origin/changeset-release/main || echo "changeset-release/main doesn't exist yet"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)